README.md

```diff
@@ -14,7 +14,7 @@ Only a subset of BLAS is available under `nimblas`, with more operations added
 on necessity.
 
 For a higher-level linear algebra library based on this, check out
-[neo](http://unicredit.github.io/neo/).
```

Comment:
This link was not working so I thought that Neo was discontinued. I now realize that it still exists. That being said I think it probably makes more sense to link to Arraymancer instead (or to both Arraymancer and Neo), given that Arraymancer is the most popular nim tensor library.

Reply:
We should link to both.